### PR TITLE
Forward logs to Monitoring

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 1]
```diff
@@ -88,6 +88,21 @@ There are two levels of filters to help drill down on the logs to make visible:
...
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Runtime Fabrics* in the sidebar navigation.
+. Select the Runtime Fabric based on the name used during installation.
+. From the *Log Forwarding* tab, select *Forward application logs to Anypoint Monitoring* checkbox.
+. Click *Apply changes* to confirm the changes.
+
+[NOTE]
+====
+* The *Log Forwarding* tab is hidden if the Runtime Fabric version is earlier than version 1.4.35.
```
Comment (yianL) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: `1.4.35` is not a public-facing version. Maybe `1.4.44`?

**Review by yianL — APPROVED** [review]

**Comment by paigehargrave** [comment]
@yianL - Thank you!

File: modules/ROOT/pages/runtime-fabric-logs.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: Updated - thank you!

**Review by Cristian-Venticinque — APPROVED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 2]
```diff
@@ -88,6 +88,21 @@ There are two levels of filters to help drill down on the logs to make visible:
...
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Runtime Fabrics* in the sidebar navigation.
+. Select the applicable Runtime Fabric based on the name used during installation.
+. Select the *Log Forwarding* tab, and then select the *Forward application logs to Anypoint Monitoring* checkbox.
+. Click *Apply changes* to confirm the changes.
+
+[NOTE]
+====
+* The *Log Forwarding* tab is hidden if the Runtime Fabric version is earlier than version 1.4.44.
```
Comment (phuang2019) [inline modules/ROOT/pages/runtime-fabric-logs.adoc:103]: Jeff asked me to update the supported version to be > 1.4.54 recently.

Comment (yianL) [reply modules/ROOT/pages/runtime-fabric-logs.adoc:103]: Got it, can you send a PR to update this value?

Comment (paigehargrave) [reply modules/ROOT/pages/runtime-fabric-logs.adoc:103]: @phuang2019 @yianL - Thank you! PTAL at https://github.com/mulesoft/docs-runtime-fabric/pull/188.

**Pull request merged**